examples/threadExample/src/main/java/io/realm/examples/threads/ThreadFragment.java

```diff
@@ -161,8 +158,8 @@ public void onPause() {
     }
 
     @Override
-    public void onDestroy() {
-        super.onDestroy();
+    public void onStop() {
+        super.onStop();
```

Comment:
I am not against using `onStart`/`onStop` instead of `onCreate`/`onDestroy` but this differs from other the other examples. So these should be updated as well. See also https://github.com/realm/realm-java/issues/874